bundles/plugins/org.bonitasoft.studio.contract/src/org/bonitasoft/studio/contract/ui/wizard/ContractInputGenerationWizard.java

```diff
@@ -75,7 +74,8 @@ public ContractInputGenerationWizard(final ContractContainer contractContainer,
             final FieldToContractInputMappingOperationBuilder operationBuilder,
             final FieldToContractInputMappingExpressionBuilder expressionBuilder,
             final IPreferenceStore preferenceStore,
-            final ISharedImages sharedImagesService) {
+            final ISharedImages sharedImagesService,
+            final ContractInputGenerationInfoDialog infoDialog) {
```

Comment:
in order to mock it, because with changes I made, some tests failed. As you can't spy the wizard I didn't find other solution